src/components/FeedbackPage/FeedbackPage.scss

```diff
@@ -0,0 +1,31 @@
+@import '~style/variables';
+
+.feedbackWrapper {
+  @include container;
+
+  padding: 48px 0;
+  background: $color-overview--020;
+}
+
```

Comment:
I think there should be a wrapper, with the background colour, and a nested container (with max-width). Otherwise the colour doesn't go to the edge of the page and looks weird.

![image](https://user-images.githubusercontent.com/4443482/32812627-275a66c4-c9fa-11e7-9621-e0cc004cf5f4.png)